tests/pyuntl_test.py

```diff
@@ -148,7 +148,7 @@ def testCreatesCreator(self):
 
     def testCircularEquality(self):
         self.assertEqual(
-            py2dict(untlxml2py(StringIO.StringIO(
+            py2dict(untlxml2py(BytesIO(
```

Comment:
What happens if we use `StringIO` instead of `BytesIO`?

Reply:
pydict2xmlstring function returns bytes. So I used BytesIO. I'll get a TypeError if I use StringIO instead.